src/lib/message_filters.py

```diff
@@ -30,3 +31,22 @@ def filter_time_range(messages, time_key, start_time, end_time):
     @staticmethod
     def filter_noise(messages, message_key, noise_fn):
         return [td for td in messages if not noise_fn(td.get(message_key))]
+    
+    @staticmethod
+    def subsample_messages(messages):
+        '''
+        Generates sample messages for labelling in coda
```

Comment:
This description feels over-specific (it's plausible that you might want to subsample the data for purposes other than Coda)